### ISSUE TITLE
UIP3 - Marketing Budget Proposal

### ISSUE THREAD

**Issue by Bullchad123** [ubiq/UIPs#3]
The purpose of this document is to lay out the general guidance for the marketing budget, documenting funds available and their intended purpose of use.
 
**General Approach and priorities** 

![image](https://user-images.githubusercontent.com/29441296/46034207-fedab000-c0f7-11e8-93df-f7f07233ded9.png)

Our marketing approach will follow two stages:

First, we will focus on completing the Ubiq branding for Escher, Twine and providing marketing materials for use on twitter and other social media sites. We will also look to establish a regular translation of our Quarterly reports into key languages for our supporters.

We will achieve this through bounties and competitions where possible. If we cannot source this via the community, then we will utilise third parties on a per-project basis.

The second stage will involve the funding of new research and marketing materials being created, posting to third party news sites (paid if needed), trialling some targeted advertising and the sponsorship of podcasts and YouTube channels. 

**Marketing Budget**

We request that a regular drawdown of 510 UBQ is taken from the marketing fund every 28000 Blocks (approx. every 4 weeks) starting from the moment this voting period concludes. We plan to use this as laid out in our general approach, however, to remain flexible we also will reserve the right to spend this at our discretion. We may build the funds up over a period of months or to use it ad hoc as requirements present themselves. 

We will report on our activities with these funds on our quarterly report, providing a high-level overview of expenditure and accountability to the community. Suggestions are welcome in the #ubiq-marketing channel of our discord.

Kris Lester will ultimately take ownership for these funds and will make final decisions on how these are utilized. Funds drawn down will sit in a multisig wallet, with 3 members of the community/core team (including Kris) having access.

For transparency, the address for the marketing budget will be included in the Escher vote and all expenditure will be accounted for in the quarterly reports.
 
**NDF and Marketing Fund Clarification**

![image](https://user-images.githubusercontent.com/29441296/46034310-39dce380-c0f8-11e8-9238-ee333aa27fbf.png)
_Figures not accurate as UIP2 is yet to be implemented_

As the above picture highlights, there are two different sections for the resources available to the Ubiq team. For clarity we will only draw down from the marketing fund and not the NDF for these activities.

The NDF fund will be utilized for network development bounties and managed by Julian Yap, Luke Williams and Alex Sterk. 
 
**Large purchases**

This proposal allows the team to make quick decisions around marketing without the requirement to run everything through the governance system. However, besides the proposal, there might be a future requirement for additional large purchases which fall outside this budget.

An example of these items might be a one-off payment to integrate a merchandise store into our website for $10k or sponsoring an event for $50-70k.
We will put these purchases through the Escher system as a separate vote and put to the community for feedback.
 
**Future Changes to the Marketing Budget**

Whilst this proposal sets out a modest budget of 510 UBQ per 28000 blocks, there may be a need to change this, depending on new requirements. We will put any alterations through the governance system and consult the community for approval. 
 
**Voting Period**

Propose that we run this over a voting period of 14000 blocks with a minimum participation threshold of 20%.

**Comment by ElevenXi** [comment]
Moving forward, I think this is a fantastic and needed idea.

**Comment by sambiohazard** [comment]
Hey all, finally made an account. Ok so here are some questions i have.

1. What does branding means/includes? e.g. Logos, website design, infographics, physical collectibles like stickers, coins, hats!! etc.

**Comment by sambiohazard** [comment]
Ok thanks for the clarification. Here is what i think about this proposal. I am excited that someone is taking up this difficult task. Thanks for doing this! Now the thoughts & suggestions

1. Branding: Please don't make it a one time thing like logos etc, make it a long term persistent thing that eventually is present everywhere uniformly within all parts of the project. Suggestion for this is to start light and experimental with one or two things but keep using them consistently in social media and your terminology while talking with media outlets/podcasts. Also it would be nice if a consistent color palette is chosen for most branding material. Most important thing here is to not hype these things as a big one time event but to subtly introduce them as you communicate over different channels, so as to not make it look like shitcoin pumpimg. It creates a very bad impression (creates expectation & fails) and drives away any serious investors. I like the idea of using professional third parties for this if needed.

2. Community Engagement: Same here, please do not make it about bounties/competition/hackathons (these can be covered with NDF probably, but i would rather prefer dev meetups like SF/London dev meetups in bitcoin, than hackathons) only. Such will only attracts individuals who are interested in money & fame and most probably for short time. Suggestion for this is to do activities more along the lines of what Kris did with his charity site, i.e. more meaningful long term impact projects and community meetups. This creates a expectation that you are to participate and contribute because the thing you are doing is meaningful to you and not for fame/money. Also removes any expectations of profits/pumps. Some examples can be creating tutorials, answering questions on reddit/discord, doing meetups, going to local town events and talk to people there, maybe give a presentation.

3. Research: I am not sure about research but i am assuming it means something like what Whiteblock(?) did with comparison of ETH & UBQ. It would be awesome to have more of that although i think paid research is neither good nor within budget. Maybe we can contact such firms and offer any help from our side if and when they want to do a study, probably have someone dedicated for this role, this might come in form of Community Engagement mentioned above.

4. News Outreach: Again sound like a fantastic idea if done right. This can be done in 2 ways.
i) talk about what is already done, how it came about, rationale behind decisions etc (do not forget to use that sweet sweet branding here) and the long term goal/vision. This will be excellent material for new investors.
ii) Do release of a feature or of quarterly report and then reach out to various news outlets & podcast to highlight that release in the news cycle for next 2 weeks to get maximum visibility for work done. This is sorely needed for UBQ i think. This way we will ditch the "announcement of an announcement" & pump and dump and relieve any pressure for completion of goals while also keeping the news cycle and exposure going.

5. Advertisement & Sponsorship: I don't think crypto advertising works, too much bad ads out there, so bad company to be in when trying to gain good reputation. It probably puts you into more ignore lists than "lets check it out" lists. Again i think podcasts/news articles as suggested above work much better for this.
As for sponsorship, i think we need to be selective about this and sponsor long standing projects which share our values instead of anything we can get our hands on, especially avoid high profile meetups/conferences IMO. Some suggestions are fediverse projects like Mastodon, Peertube, crypto projects like DEXes, sponsor town level project that help you local community, heck even EFF if we have budget for those. Also tie this in with News Outreach once sponsorship is in place, maybe a podcast appearance with representatives of both projects.

6. Marketing Budget & Ownership: The budget seems decent for getting started although probably should be ramped up gradually to 2x if projects succeed(already mentioned in the proposal, so great!). Also would be great to have multisig setup with 2 of 3 for now for ownership. Its not an issue of trust but of security and availability. Also distributes the responsibility. Nice idea to have reporting in quarterly, maybe followed by a ESCH vote for further funds/spending.

7. Large purchases & Voting: Seems all good to me, nothing to add here.

Once again thanks for putting in crucial work and making this proposal. I hope you find this feedback useful, probably most of it you already thought about. Nice to have a small contribution in shaping this. Thanks for the opportunity :)
Looking forward to lots of exposure!

**Comment by krislester123** [comment]
Hey,

Great feedback!

For clarity, we had an initial discussion around the NDF and we decided that it would be best to keep that separate from logo designs etc. I'm open to opposing arguments here though. 

'Research' is a new section on the Ubiq blog, more details to be released about this soon.

Agree with you on news outreach and advertisements. We will be running a small trial on adverts separate to this proposal based on donations from community members (more details to be declared in the monthly report in a few days). We will report back to members on this trial so we can make informed decisions about whether buying ad-space is worth it or not.

Again, agree with you on Marketing Budget and Ownership. We are drawing down a small amount to start off with, as the main marketing fund will need to be replenished at a faster rate than what we draw down. Once we get some success we can push through another vote to expand the scope. Regarding ownership of funds, I won't be able to touch the main marketing fund (currently 14,000 UBQ), only the UBQ being drawn off per month. I'll have a word with Sterky about the possibility of a multisig setup, but it needs the flexibility of me being able to make quick decisions if they present themselves. So I'll get back to you on that one.

**Comment by krislester123** [comment]
OK, so I've updated the proposal to include a multisig setup for the funds being drawn down.
